src/librc/librc.c

```diff
@@ -304,6 +306,8 @@ detect_container(const char *systype RC_UNUSED)
 		return RC_SYS_SYSTEMD_NSPAWN;
 	else if (exists("/.dockerenv"))
 		return RC_SYS_DOCKER;
+	else if (exists("/run/.containerenv"))
+		return RC_SYS_PODMAN;
```

Comment:
Please move this above the /.dockerenv check. See systemd for reference.

https://github.com/systemd/systemd/blob/v255/src/basic/virt.c#L640
https://github.com/systemd/systemd/blob/v255/src/basic/virt.c#L764-L767